LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/authentication/LabApiAuthenticationClient.java

```diff
@@ -72,6 +76,45 @@ public LabApiAuthenticationClient(@NonNull final String labSecret, final String
 
     @Override
     public String getAccessToken() throws LabApiException {
+        return getAccessToken(DEFAULT_ACCESS_TOKEN_RETRIES);
+    }
+
+    public String getAccessToken(final int retries) throws LabApiException {
+
+        // Do this in a loop, if we get an exception or null result, try again
+        for (int i = 0; i < retries; i++) {
+            System.out.printf(Locale.ENGLISH, "getAccessToken attempt #%d%n", (i + 1));
```

Comment:
Could consider starting the loop from `i = 1` and until `i <= retries` then you wouldn't need to add/subtract from i when printing

Reply:
Updated to reflect this